lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/modal/ModalStack.java

```diff
@@ -142,6 +143,9 @@ public boolean handleBack(CommandListener listener, ViewController<?> root) {
     public boolean isEmpty() {
         return modals.isEmpty();
     }
+    public List<ViewController<?>> getChildren(){
+        return new LinkedList<>(modals);
```

Comment:
this has to be a LinkedList?

Reply:
Lightweight list is a linkedlist, especially for iteration not index access, List is an interface in java, and this is a "View".